prow/prowjobs/google/exposure-notifications-verification-server/en-verification-server.yaml

```diff
@@ -33,3 +33,32 @@ presubmits:
           requests:
             cpu: 7
             memory: '16Gi'
+  - name: pull-en-verification-server-release-e2e-canary
+    cluster: build-apollo-server
+    always_run: true
```

Comment:
I thought we werent running this all the time and were going to do a fixed interval instead?

Reply:
Sorry forgot to mention, this is for faster turnaround on PRs. Once the PR confirmed working I'll set the periodic runs right away

Reply:
Will this fail our existing PRs though? Is there a way to mark it as "allow failure"?

Reply:
Yeah good catch, changed it to not always run

Reply:
Also set optional to true so it's not blocking